x-pack/plugins/security_solution/common/endpoint/types/actions.ts

```diff
@@ -21,7 +24,8 @@ export interface EndpointAction {
   };
 }
 
+export type HostIsolationRequestBody = TypeOf<typeof HostIsolationRequestSchema.body>;
+
 export interface HostIsolationResponse {
   action?: string;
-  message?: string;
```

Comment:
if we're using this type to only refer to success, then we can drop the `?` on action